src/Hng.Application/Features/Products/Dtos/ProductDto.cs

```diff
@@ -2,8 +2,13 @@ namespace Hng.Application.Features.Products.Dtos
 {
     public class ProductDto
     {
-        public Guid id { get; set; }
-        public string name { get; set; }
-        public string description { get; set; }
+        public Guid Id { get; set; }
```

Comment:
Use `JsonPropertyName` attribute to turn all your request and response dtos to `snake_case`